packages/tokens/build.js

```diff
@@ -115,6 +120,12 @@ StyleDictionary.registerFilter({
   matcher: (token) => token.attributes.category.includes('spacing'),
 })
 
+/** Remove VADS web font tokens */
+StyleDictionary.registerFilter({
+  name: 'filter/fonts/mobile-fonts',
+  matcher: (token) => token.attributes.category !== 'vads-font',
```

Comment:
Not sure if there's another way without digging at all, but just noting it feels weird to have a general filter that needs to be applied to all files that didn't previously have a filter to get rid of the `vads-font`s.

Reply:
There doesn't seem to be a way to apply filters to an entire platform or transform group, but style-dictionary 4.0 introduces preprocessors which might be able to clean this up. Created https://github.com/department-of-veterans-affairs/va-mobile-library/issues/604 to follow up on it after we update to 4.0.